File: core.shared/Typescript/DTO/V1/Participant/choixAirbus.ts
/* Auto Generated */

import { EnuContrat } from "./../../../Enumerations/V1/enuContrat";
import { EnuContratAction } from "./../../../Enumerations/V1/enuContratAction";
import { EnuRaisonAdhesionParticipant } from "./../../../Enumerations/V1/enuRaisonAdhesionParticipant";
import { EnuContratResiliation } from "./../../../Enumerations/V1/enuContratResiliation";

export interface ChoixAirbus {
    id: number;
    numParticipant: string;
    contratParticipant: EnuContrat;
    contratAction: EnuContratAction;
    dateEffet?: Date;
    raisonAdhesion?: EnuRaisonAdhesionParticipant;
    libelle: string;
    description: string;
    dateResiliation?: Date;
    dateAdhesion?: Date;
    contratResiliation: EnuContratResiliation;
}

File: core.shared/Net/DTO/V1/Participant/ChoixAirbus.cs
using Core.Shared.Net.Enumerations.V1;

namespace Core.Shared.Net.DTO.V1.Participant
{
    public class ChoixAirbus
    {
        public int id { get; set; }
        public string NumParticipant { get; set; }
        public EnuContrat ContratParticipant { get; set; }
        public EnuContratAction ContratAction { get; set; }
        public DateTime? DateEffet { get; set; }
        public EnuRaisonAdhesionParticipant? RaisonAdhesion { get; set; }
        public string Libelle { get; set; }
        public string Description { get; set; }
        public DateTime? DateResiliation { get; set; }
        public DateTime? DateAdhesion { get; set; }
        public EnuContratResiliation ContratResiliation { get; set; }
    }
}
